### DISCUSSION TITLE
#238 I wanted to ask what software is @lucasavila00 using in this image to run mistral and see the layers inside of it ( looks like a gpu profiler)

### DISCUSSION THREAD

**Discussion by xXWarMachineRoXx** [EricLBuehler/mistral.rs#248] [Q&A] [answered]
![image](https://github.com/EricLBuehler/mistral.rs/assets/47086987/ed46d520-2bf8-416d-8f04-773277fcf8ba)

#238 @lucasavila00 , I just found your PR interesting and thought it be great I also could profile my gpu for layers when running llms.
I know its not that difficult of a question but I didn't know where else I could ask that 😅 

- So basically, what software did you use to profile your gpu like that?

**Comment by EricLBuehler** [comment]
@xXWarMachineRoXx , I think he uses Nvidia profiler.

**Reply by xXWarMachineRoXx** [reply]
thanks for answering!

**Comment by lucasavila00** [comment, marked as answer]
@xXWarMachineRoXx latest nvidia profiler

![image](https://github.com/EricLBuehler/mistral.rs/assets/12750442/a9577a52-de5d-438c-aa78-903cd5b19874)

**Reply by xXWarMachineRoXx** [reply]
Thx! Much appreciated!!
